src/diffpy/pdfgui/gui/pdfguiglobals.py

```diff
@@ -31,7 +32,7 @@
 isAltered = False
 
 # Resolve APPDATADIR base path to application data files.
-_mydir = os.path.abspath(resource_filename(__name__, ""))
+_mydir = os.path.abspath(str(resources.files(__name__).joinpath("")))
```

Comment:
whenever we touch lines of code that do path munging, switch the code to using pathlib.Path.  Then do it for the full module.  This also on a separate PR because it will be harder to merge.